components/security/secure_tools.rst

```diff
@@ -5,6 +5,12 @@ The Symfony Security component comes with a collection of nice utilities
 related to security. These utilities are used by Symfony, but you should
 also use them if you want to solve the problem they address.
 
+.. note::
+
+    The functions described in this article were introduced in PHP 5.6 or 7.
+    For older PHP versions, a polyfill is provided by the
+    `Symfony Polyfill Component`_.
```

Comment:
We need a component section for the polyfills to which we can link instead where explain that you should rather install the polyfill packages you need instead of the big meta package imo.